### DISCUSSION TITLE
VIN Display

### DISCUSSION THREAD

**Discussion by natrlhy** [teslamate-org/teslamate#2895] [Ideas]
Is there a dashboard that displays the VIN of the vehicle? I'm not seeing it displayed anywhere and would like to see it without opening up the Tesla App just to get it.

**Comment by cwanja** [comment]
It is in the database, so you could query is from Grafana. But it is not displayed on any dashboard in any format - full or last six.

**Comment by DrMichael** [comment]
Not sure, whether it really makes sense, because that is very static data.
One possibility would be to include it in the car selection:
![image](https://user-images.githubusercontent.com/8087682/189860127-f6ee485f-d4b8-4d2c-9e16-9a5f05574edb.png)

**Reply by natrlhy** [reply]
How could this be done?

**Reply by natrlhy** [reply]
I see the variable in the dashboard, but now I'm trying to figure out how you displayed the VIN next to the `name`
Query:
`SELECT name,vin AS __text, id AS __value FROM cars ORDER BY display_priority ASC, name ASC;`

**Reply by natrlhy** [reply]
Ah I think I got it!
`SELECT CONCAT (name, ' ', '(',vin,')') as __text, id AS __value FROM cars ORDER BY display_priority ASC, name;`

**Reply by natrlhy** [reply]
Create [PR](https://github.com/adriankumpf/teslamate/pull/3062)
